fearless/Modules/Staking/StakingAmount/StakingAmountInteractor.swift

```diff
@@ -113,34 +113,20 @@ final class StakingAmountInteractor {
     }
 
     private func provideMinimumAmount() {
-        let factoryOperation = runtimeService.fetchCoderFactoryOperation()
-
-        let minimumOperation = PrimitiveConstantOperation<BigUInt>(path: .existentialDeposit)
-        minimumOperation.configurationBlock = {
-            do {
-                minimumOperation.codingFactory = try factoryOperation.extractNoCancellableResultData()
-            } catch {
-                minimumOperation.result = .failure(error)
-            }
-        }
+        let wrapper = eraInfoOperationFactory.networkStakingOperation()
```

Comment:
This this too heavy for such use case. Staking info fetches other information which is not necessary. To simplify constants fetching there is ```RuntimeConstantFetching```. Please, try to use this guy.